python/ray/tests/test_state_api.py

```diff
@@ -1570,7 +1571,22 @@ def verify_output(cmd, args: List[str], necessary_substrings: List[str]):
     sys.platform == "win32",
     reason="Failed on Windows",
 )
-def test_list_get_actors(shutdown_only):
+@pytest.mark.parametrize(
+    "override_url",
+    [
+        None,
+        "https://external_dashboard_url",
+        "https://external_dashboard_url/path1/?query_param1=val1&query_param2=val2",
+        "new_external_dashboard_url",
+    ],
+)
+def test_list_get_actors(shutdown_only, override_url):
```

Comment:
i think we could duplicate the original test, and rename it to something more representative of what's this doing. e.g. `test_state_api_with_external_dashboard_override`.

Reply:
When `override_url` is set to `None` this should be the same as the original test. Does it still make sense to duplicate?

Reply:
I think so - since there are also `override_url` values right?

Reply:
ok, updated